File: 1929-concatenation-of-array/1929-concatenation-of-array.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int ansLen = 2 * nums.Length;
        int[] ans = new int[ansLen];
        
        for (int i = 0; i < ansLen; i++)
            ans[i] = nums[i % nums.Length];
        return ans;
    }
}

File: 1929-concatenation-of-array/1929-concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int ansLen = 2 * nums.length;
        int[] ans = new int[ansLen];
        
        for (int i = 0; i < ansLen; i++)
            ans[i] = nums[i % nums.length];
        return ans;
    }
}